docs/topics/proc-configure-launcher-sso-realm.adoc

```diff
@@ -0,0 +1,18 @@
+[[launcher-sso-settings]]
+= Configuring {launcher} {RHSSO} Federated Identity Settings
+In order to use {launcher} in {OpenShiftOnline}, you must grant it access to your GitHub and OpenShift accounts. This is a one-time action.
```

Comment:
This looks like a material for an admonition, not really an introduction to a topic. The introduction should contain information about whom (if not obvious), why, and how performs the procedure.